Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -224,7 +230,16 @@
 if ~isfield(x.Q.BASIL,'bMasking') || isempty(x.Q.BASIL.bMasking)
 		fprintf('BASIL: Setting default option bMasking = true\n');
 		x.Q.BASIL.bMasking = true;
-	end
+end
+
+if nargin<7 || isempty(bMergingSessions)
+	bMergingSessions = 0;
+end
+
+% If we need to merge sessions, we will use Fabber
+if bMergingSessions
+    bUseFabber = 1;
```

Comment:
Please can you separate session merging and the use of FABBER. E.g., Mervin will use session merging for all Singapore studies, where most are without DEBBIE.